init.lua

```diff
@@ -424,12 +424,16 @@ end
 --
 --  Add any additional override configuration in the following tables. They will be passed to
 --  the `settings` field of the server config. You must look up that documentation yourself.
+--
+--  If you want to override the default filetypes that your language server will attach too you can
```

Comment:
Sorry to be a pest, but you want 'attach to' not 'attach to' otherwise looks great!